ckanext/unaids/theme/templates/package/activity.html

```diff
@@ -0,0 +1,76 @@
+{% extends "package/read_base.html" %}
+
+{% block subtitle %}{{ _('Activity Stream') }} {{ g.template_title_delimiter }} {{ super() }}{% endblock %}
+
+{% block primary_content_inner %}
+<h1 class="hide-heading">
+  {% block page_heading %}
+  {{ _('Activity Stream') }}
+  {% endblock %}
+</h1>
+{% snippet 'snippets/activity_stream.html', activity_stream=activity_stream, id=id, object_type='package' %}
+
+<link rel="stylesheet" href="/bootstrap-table.css" />
+<div id="ActivityStreamLoading" class="text-center text-muted">
+  <h2>
+    <i class="fa fa-spinner fa-spin"></i>
+    <span> {{ _('Loading') }}</span>
+  </h2>
+</div>
+<div id="ActivityStreamContainer">
+  <table class="activity">
+    <thead>
+      <tr>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody></tbody>
+  </table>
+</div>
```

Comment:
the div id="ActivityStreamLoading" is the actual extension of this template.
You don't have to copy the whole file over. With help of ckan jinja extension mechanism you can do it in a nicer way (without core code duplication).
Take a look at: https://docs.ckan.org/en/2.9/theming/templates.html#extending-templates-with-ckan-extends
(leave the block `primary_content_inner` intact and add your pagination html separatly, sth like:
```
{% ckan_extends %}
<div id="ActivityStreamLoading">
[...]
```